### ISSUE TITLE
Can't access Web UI anymore (after update?)

### ISSUE THREAD

**Issue by onlineaddy** [binhex/arch-delugevpn#196]
I think delugevpn is running, but I can no longer access the Web UI. I'm not sure if this is since I updated to the latest image. Any help is much appreciated.

My supvervisord.log is on [pastebin](https://pastebin.com/cJ05iaez).

**Comment by binhex** [comment]
is this correct?, taken from your log:-
```
2020-09-15 04:03:53.562724 [info] LAN_NETWORK defined as '192.168.1.0/24'
```

see Q4 here for help with it:- https://github.com/binhex/documentation/blob/master/docker/faq/vpn.md

**Comment by onlineaddy** [comment]
Yes, LAN_NETWORK should be correct: the host IP is 192.168.1.9 and the subnet mask is 255.255.255.0.

**Comment by onlineaddy** [comment]
Any other idea what else I should check? I didn't change any of my settings, other than updating the image. (But, I'm not 100% certain things stopped working because of the update.)

**Comment by onlineaddy** [comment]
Also, if I wanted to roll back the version to test, which version would you recommend?

**Comment by onlineaddy** [comment]
What about Python version?
